File: RUBY/actividades/actividad 24-05.rb
puts"Ingrese nombre:"
n = gets.chomp
puts"Ingrese apellido:"
ap = gets.chomp
puts"Numero 1:"
n1 = gets.chomp.to_i
puts"Numero 2:"
n2 = gets.chomp.to_i
puts"Numero 3:"
n3 = gets.chomp
sum = n1 + n2 + n3.to_i
prom = sum / 3

puts("Tu nombre completo es #{n} #{ap}")
puts("El promedio es: #{prom}")

File: DART/actividades/actividad 24-05.dart
import 'dart:io';

void main() {
  print("Ingrese nombre:");
  String? n = stdin.readLineSync();
  print("Ingrese apellido:");
  String? ap = stdin.readLineSync();
  print("Numero 1:");
  int n1 = int.parse(stdin.readLineSync()!);
  print("Numero 2:");
  int n2 = int.parse(stdin.readLineSync()!);
  print("Numero 3:");
  int n3 = int.parse(stdin.readLineSync()!);

  int sum = n1 + n2 + n3;
  double prom = sum / 3;

  print('tu nombre completo es ${n} ${ap}');
  print('el promedio es: ${prom}');

}